src/contains.js

```diff
@@ -1,15 +1,13 @@
 define(function() {
-
   /**
    * contains checks to see if a string contains another string
    *
    * @access private
    * @function contains
    * @param {string} str - The string we want to check for substrings
    * @param {string} substr - The substring we want to search the first string for
-   * @returns {boolean}
+   * @returns {boolean} true if a string contains another string
```

Comment:
it returns wether or not a string contains another string. current wording is ambiguous in the falsey case